src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py

```diff
@@ -315,7 +315,6 @@ def add_to_nwbfile(
 
                 metadata['Ecephys']['ElectricalSeries'] = dict(name=my_name, description=my_description)
 
-        The default is False (append mode).
```

Comment:
@CodyCBakerPhD it seems that it came from here. Let me test, anyway, this was an error.

Reply:
You would be correct!

https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py#L318 would be interpreted as an argument not the description of the argument it is attached to